docs/reference/ingest/search-inference-processing.asciidoc

```diff
@@ -102,10 +102,13 @@ Here, you'll be able to:
 1. Choose a name for your pipeline.
   - This name will need to be unique across the whole deployment.
   If you want this pipeline to be index-specific, we recommend including the name of your index in the pipeline name.
+  - If you do not set the pipeline name, a default unique name will be provided upon selecting an ML trained model.
```

Comment:
If the user doesn't touch the pipeline name field, then upon selecting a model, the name will be automatically generated using the `ml-inference-<index name>-<normalized model name>` pattern. Normalization of the model name means every non-alphanumeric symbol other than "-" is converted to underscore.

But if the pipeline name is edited, selecting a model does not change it.

<img width="720" alt="Screenshot 2023-12-28 at 11 54 34" src="https://github.com/elastic/elasticsearch/assets/14224983/be50f2f9-0b06-49f4-94d9-4ebdc9c9966a">